packages/suite/src/components/firmware/ReconnectDevicePrompt.tsx

```diff
@@ -41,15 +41,22 @@ const Wrapper = styled.div`
     }
 `;
 
-const Content = styled.div`
+const Content = styled.div<{ isCentered: boolean }>`
     display: flex;
     flex-direction: column;
     padding: 10px 14px;
     margin-left: 24px;
 
-    ${variables.SCREEN_QUERY.MOBILE} {
-        margin-left: 0;
-    }
+    ${({ isCentered }) =>
+        isCentered
+            ? css`
+                  margin: 0 auto;
+              `
+            : css`
+                  ${variables.SCREEN_QUERY.MOBILE} {
+                      margin-left: 0;
```

Comment:
I would flip this. ABOVE_MOBILE add margin-left: 24px, otherwise no margin-left.

Or better:
Maybe we can just use `justify-content: space-around;` on the wrapper and we will not need those margins at all, I guess

Reply:
0041d13bc91ffbff202dc1fdbc6e26c09c8c12ac